WebClient/src/core/modelCollection.js

```diff
@@ -39,18 +39,21 @@ var ModelCollection = L.Evented.extend({
 
         this.set = function (newModels) {
             var removedModels = difference(models, newModels);
+            var addedModels = difference(newModels, models);
             models = toArray(newModels);
             
             this.fire('remove', { models: removedModels });
+            this.fire('add', { models: addedModels });            
             this.fire('change', { models: models });
         };
 
         this.add = function (newModels) {
```

Comment:
Propose rename: newModels -> addModels, it is not the complete list

Reply:
in this case: 
- newModels is a new list of models that should override models field.
- addedModels is a list of models that were introduced by the newModels list and were not present in the models list.